sdk/ml/azure-ai-ml/azure/ai/ml/entities/_assets/_artifacts/code.py

```diff
@@ -73,18 +73,21 @@ def _load(
         data: Optional[Dict] = None,
         yaml_path: Optional[Union[PathLike, str]] = None,
         params_override: Optional[list] = None,
-        **kwargs,
+        **kwargs: Any,
     ) -> "Code":
         data = data or {}
         params_override = params_override or []
         context = {
             BASE_PATH_CONTEXT_KEY: Path(yaml_path).parent if yaml_path else Path("./"),
             PARAMS_OVERRIDE_KEY: params_override,
         }
-        return load_from_dict(CodeAssetSchema, data, context, **kwargs)
+        res: Code = load_from_dict(CodeAssetSchema, data, context, **kwargs)
+        return res
 
     def _to_dict(self) -> Dict:
-        return CodeAssetSchema(context={BASE_PATH_CONTEXT_KEY: "./"}).dump(self)  # pylint: disable=no-member
+        # pylint: disable=no-member
+        res: dict = CodeAssetSchema(context={BASE_PATH_CONTEXT_KEY: "./"}).dump(self)
```

Comment:
This is a great example of what I was recommending you do in those other examples